index.mk

```diff
@@ -36,6 +41,15 @@ ci: verify test whitesource
 # Install tasks
 # -------------
 
+# Update the .env file with secrets from Vault (https://github.com/Financial-Times/vault/wiki/)
+.env:
+	@if [[ -z "$(shell command -v vault)" ]]; then echo "Error: You don't have Vault installed. Follow the guide at https://github.com/Financial-Times/vault/wiki/Getting-Started"; exit 1; fi
+	@if [[ -z "$(shell find ~/.vault-token -mmin -480)" ]]; then echo "Error: You are not logged into Vault. Try vault auth --method github."; exit 1; fi
+	@if [[ -z "$(shell grep .env .gitignore)" ]]; then echo "Error: .gitignore must include .env"; exit 1; fi
```

Comment:
Should `--silent --no-messages` be added to the `grep` here like it is on line 49 below?

Reply:
The changes below below allow `npmignore` to be omitted. I think I'm happy to leave the output if `.gitignore` is not present -- with the assumption git is used.